examples/fabric-admin/commands/fabric-sync/FabricSyncCommand.h

```diff
@@ -19,6 +19,16 @@
 #pragma once
 
 #include <commands/common/CHIPCommand.h>
+#include <commands/pairing/OpenCommissioningWindowCommand.h>
+#include <commands/pairing/PairingCommand.h>
+
+constexpr uint32_t kSetupPinCode               = 20202021;
```

Comment:
Where is this used?

Reply:
It is used in bridge management, removed from this PR